internal/podman/client.go

```diff
@@ -383,6 +394,29 @@ func (o *client) ContainerLogs(ctx context.Context, name string) ([]string, erro
 	return logs, nil
 }
 
+func (o *client) ContainerUptime(ctx context.Context, name string) (time.Duration, error) {
```

Comment:
Instead of creating these functions can we reuse ContainerInspect?

Reply:
We could, but we don't need it for docker. I wanted to avoid having to implement things in multiple places if we don't use them in both.

Reply:
Well container inspect is implemented for podman and it does not impact docker, check https://github.com/mongodb/mongodb-atlas-cli/pull/3081/files#diff-b2a128f7fb3a56f3cca07064286b8b276642d57d2b75dfa8b1924448b1888a15R152 it is just a matter of adding more fields to the podman only internal struct

Reply:
Container inspects only deserializes the JSON into a struct. But doesn't do any parsing.

If I add the uptime to the inspect struct, I would have to do one of the following:
- Parse the created at the level of the caller
- Change the current logic of the `ContainerInspect` function and use two separate structs, one for the raw JSON we get back and one for the parsed JSON, and do the conversion from a string to a DateTime in the inspect method.

Both options are worse than what we have now in my opinion.

Reply:
you can do `o.runPodman(ctx, "inspect", "--format", "[\"{{.State.Status}}\", \"{{.State.StartedAt}}\"]", name)` for instance

Reply:
updated